cmd/create.go

```diff
@@ -244,6 +244,12 @@ var createCmd = &cobra.Command{
 		fmt.Println(endingMsgStyle.Render("\nNext steps:"))
 		fmt.Println(endingMsgStyle.Render(fmt.Sprintf("• cd into the newly created project with: `cd %s`\n", project.ProjectName)))
 
+		if options.Advanced.Choices["Tailwind"] {
+			options.Advanced.Choices["Htmx"] = true
+			fmt.Println(endingMsgStyle.Render("• Install the tailwind standalone cli if you haven't already, grab the executable for your platform from the latest release on GitHub\n"))
+			fmt.Println(endingMsgStyle.Render("• Tailwind GitHub: https://github.com/tailwindlabs/tailwindcss/releases/\n"))
```

Comment:
I would change this to point to the page explaining the Standalone CLI. That page will give more info about it and how to install it. It is still a relatively unknown feature so pointing people to an explainer page will be helpful. 
```suggestion
			fmt.Println(endingMsgStyle.Render("• More info about the Tailwind CLI: https://tailwindcss.com/blog/standalone-cli\n"))
```